packages/react/src/React.js

```diff
@@ -99,6 +100,8 @@ export {
   REACT_STRICT_MODE_TYPE as StrictMode,
   REACT_DEBUG_TRACING_MODE_TYPE as unstable_DebugTracingMode,
   REACT_SUSPENSE_TYPE as Suspense,
+  // enableHiddenAPI
+  REACT_OFFSCREEN_TYPE as unstable_Offscreen,
```

Comment:
Need to add this to `index.modern.fb.js` and `index.classic.fb.js`, too. The tests didn't fail because we don't current tests against run the compiled Facebook build artifacts, only against source.